### PR TITLE
[4.0] system update notification description

### PR DESCRIPTION
**Pull request by brianteeman**
This PR updates the description of the plugin to point to the new mail templates instead of language overrides for customisation

### CODE REVIEW

File: administrator/language/en-GB/plg_system_updatenotification.ini [hunk 1]
```diff
@@ -16,4 +16,4 @@ PLG_SYSTEM_UPDATENOTIFICATION_MAIL_MAIL_TITLE="Joomla: Update Notification"
...
 PLG_SYSTEM_UPDATENOTIFICATION_POSTINSTALL_UPDATECACHETIME_ACTION="Set it back to the default setting (6 Hours)"
 PLG_SYSTEM_UPDATENOTIFICATION_POSTINSTALL_UPDATECACHETIME_BODY="<p>In your Installer Configuration you have set the Option Update Cache (in Hours) to 0 this means that Joomla is not caching the Update. This means an email should be sent on every page visit but this is not possible. Please increase the value (6 is default) or confirm that the Joomla! Update Notification will never send you mails.</p>"
-PLG_SYSTEM_UPDATENOTIFICATION_XML_DESCRIPTION="This plugin periodically checks for the availability of new Joomla! versions. When one is found it will send you an email, reminding you to update Joomla!. Pro Tip: You can customise the email message by overriding the language string keys PLG_SYSTEM_UPDATENOTIFICATION_EMAIL_SUBJECT and PLG_SYSTEM_UPDATENOTIFICATION_EMAIL_BODY."
+PLG_SYSTEM_UPDATENOTIFICATION_XML_DESCRIPTION="This plugin periodically checks for the availability of new Joomla! versions. When one is found it will send you an email, reminding you to update. You can customise the email at System > Mail Templates.
```
Comment (richard67) [inline administrator/language/en-GB/plg_system_updatenotification.ini]: Closing quote missing at the end.

File: administrator/language/en-GB/plg_system_updatenotification.sys.ini [hunk 2]
```diff
@@ -4,4 +4,4 @@
 ; Note : All ini files need to be saved as UTF-8
 
 PLG_SYSTEM_UPDATENOTIFICATION="System - Joomla! Update Notification"
-PLG_SYSTEM_UPDATENOTIFICATION_XML_DESCRIPTION="This plugin periodically checks for the availability of new Joomla! versions. When one is found it will send you an email, reminding you to update Joomla!. Pro Tip: You can customise the email message by overriding the language string keys PLG_SYSTEM_UPDATENOTIFICATION_EMAIL_SUBJECT and PLG_SYSTEM_UPDATENOTIFICATION_EMAIL_BODY."
\ No newline at end of file
+PLG_SYSTEM_UPDATENOTIFICATION_XML_DESCRIPTION="This plugin periodically checks for the availability of new Joomla! versions. When one is found it will send you an email, reminding you to update. You can customise the email at System > Mail Templates.
```
Comment (richard67) [inline administrator/language/en-GB/plg_system_updatenotification.sys.ini]: Same as above.

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 996c8be9368c5a4b842da04ef97f23690f232b20<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33759">issues.joomla.org/tracker/joomla-cms/33759</a>.</sub>

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on 996c8be9368c5a4b842da04ef97f23690f232b20<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33759">issues.joomla.org/tracker/joomla-cms/33759</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33759">issues.joomla.org/tracker/joomla-cms/33759</a>.</sub>

**Comment by Quy** [comment]
Thanks

**Comment by brianteeman** [comment]
thanks

**Pull request merged**